.github/workflows/release.yml

```diff
@@ -26,3 +26,11 @@ jobs:
           detailed_description: |
             This is not a real lua package.
             It exists for the purpose of testing the workflow.
+  update-latest-tag:
+    runs-on: ubuntu-latest
+    name: Update latest tag
+    steps:
+      - name: Run latest-tag
+        uses: EndBug/latest-tag@latest
```

Comment:
TODO: This should fail if there's a mismatch between the current tag and `v2.x` (e.g. if the current tag is `v3.0.0`)

Reply:
Or just trigger the workflow only on `v2.*.*` tags.

Reply:
so v2.x would be a mutable tag ?

Reply:
Exactly. I always points to the latest `v2.*.*` tag. Similar to a `latest` tag, but we would stop updating it and start maintaining a `v3.x` tag if we release a `v3.*.*` version with breaking changes.

Reply:
this precludes that we wont intertwine v3 with v2 changes. Once we bump master to v3, we can't go back and port improvements to v2.x . If we want to allow for that kind of feature, we should use a branch instead, but that is more work to backport commits :/ Having a mutable tag feels weird to me but at  the same time, I know I wont backport my commits. This should be automated but do we wanna reach that level of complexity ?

Reply:
If we needed to, we could still maintain a `v2.x-backports` branch with a mutable tag that updates whenever changes are backported - but YAGNI.
Given the fact that we're not a large organisation, I think it's fine not to maintain LTS releases. I would only consider backporting from `v3` to `v2` in case of serious bugs or security issues.

It seems to be quite common for people to use the `@latest` version of GitHub actions to keep up with updates. I don't really like that though; it can lead to surprises.